src/index.ts

```diff
@@ -67,7 +67,7 @@ function activateHubExtension(app: JupyterLab, palette: ICommandPalette, mainMen
     label: 'Logout',
     caption: 'Log out of the Hub',
     execute: () => {
-      window.open(hubHost + URLExt.join(hubPrefix, 'logout'), '_blank');
+      window.location.href = hubHost + URLExt.join(hubPrefix, 'logout');
```

Comment:
This seems to be the "recommended" way to navigate to a new page in the same window. I don't know enough JS to really know, so if someone does let me know.